src/handler.rs

```diff
@@ -131,6 +131,13 @@ pub async fn handle_key_events(
                         || app.focused_block == FocusedBlock::AdapterInfos
                     {
                         app.focused_block = FocusedBlock::Device;
+                    } else if app.focused_block == FocusedBlock::Device
+                        || app.focused_block == FocusedBlock::Station
+                        || app.focused_block == FocusedBlock::AccessPoint
+                        || app.focused_block == FocusedBlock::KnownNetworks
+                        || app.focused_block == FocusedBlock::NewNetworks
+                    {
+                        app.quit();
```

Comment:
I don't think `Esc` should be used to quit the app. let's keep `ctrl +c` and `q` for that